mails/themes/dark_modern/components/layout.html.twig

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
+
+<head>
+  <title> {% block title %}Title{% endblock %} </title>
+  <!--[if !mso]><!-- -->
```

Comment:
```suggestion
  <!--[if !mso]>
```
If you want to make it work but I'm not sure this docblock is useful here the `http-equiv="X-UA-Compatible"` can be added without this comment

Reply:
Again MJML